code/modules/surgery/organs/organ.dm

```diff
@@ -115,10 +115,6 @@ var/list/organ_cache = list()
 		handle_antibiotics()
 		handle_germ_effects()
 
-	//check if we've hit max_damage
-	if(damage >= max_damage)
-		necrotize()
```

Comment:
Moving this SHOULD be OK as long as nothing ever touches the `damage` var directly, which I've seen this PR take care of